browser/admin/src/Util.js

```diff
@@ -55,7 +55,7 @@ var Util = Base.extend({
             }
         }
         var unit = 1000;
-        var units = [_(''), ('k'), _('M'), _('G'), _('T'), _('P'), _('E'), _('Z'), _('Y'), _('B')];
+        var units = ['', 'k', 'M', 'G', 'T', 'P', 'E', 'Z', 'Y', 'B'];
```

Comment:
Quick research suggest that some languages would still translate these? See e.g. <https://en.wikipedia.org/wiki/Chinese_numerals#SI_prefixes>. But you probably know what you're doing here. :-)

Reply:
`_('')` and `('k')` were wrong anyway. Thanks for the link though. I think leaving them untranslated is better that having them mistranslated by mistake.